### ISSUE TITLE
Priors for tuning

### ISSUE THREAD

**Issue by PhilippPro** [jakob-r/mlrHyperopt#16]
Can you include the possibility of priors instead of only ranges to your package? Would be awesome, although I think that it is not easy to implement this in mlrMBO...

I know that you can put in functions, but this is not super flexible.

**Comment by jakob-r** [comment]
Well it's exactly that. Just that the function would be more complex than `2^x`.
After all your param space would be `[0,1]` and the trafo would be the quantile of your prior distribution.
So pretty easy but not so easy to store in the database at the moment.

[That is the paper](https://dl.acm.org/citation.cfm?id=3220058) you have in mind, right?

**Comment by PhilippPro** [comment]
Yes, for example, probably there are also others than Jans Paper.
